components/jenkins/variables.tf

```diff
@@ -37,4 +37,26 @@ variable "key_vault_rg" {
   description = "Resource group that holds the Jenkins Key Vault"
   type        = string
   default     = "core-infra-intsvc-rg"
-}
\ No newline at end of file
+}
+
+variable "database" {
+  description = "Name of the cosmos database."
+  default     = "jenkins"
+}
+
+
+variable "cosmos_subscription_id" {
+  description = "Subscription to run against"
+  type        = string
+}
+
+variable "throughput" {
```

Comment:
I'm not sure this is working as wanted, tfplan also requesting changes for `pipeline-metrics` container, which for `throughput` requires manual destroy-apply